app/Helpers/security_helper.php

```
@@ -2,94 +2,160 @@
+use Random\RandomException;
+
+/**
+ * Replaces or inserts a single `key='value'` line in .env
+ *
+ * @param string $envKey
+ * @param string $value
+ * @return void
+ */
+function writeEnvKey(string $envKey, string $value): void
+{
+    $configPath = ROOTPATH . '.env';
```

PHPStan (2.2.7)

[error] Constant ROOTPATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)